frontend/src/views/WheresMyGene/components/HeatMap/index.tsx

```diff
@@ -170,11 +167,7 @@ export default memo(function HeatMap({
       <TopLeftCornerMask>
         <CellCountLabel>Cell Count</CellCountLabel>
       </TopLeftCornerMask>
-      <Container
-        {...{ className }}
-        id={HEATMAP_CONTAINER_ID}
-        isFmgOpen={!!cellInfoCellType || !!geneInfoGene}
```

Comment:
Do not need to check if fmg open anymore for calculating viewbox width, fixed here: https://github.com/chanzuckerberg/single-cell-data-portal/pull/4463/files#r1153767695